riak/transports/feature_detect.py

```diff
@@ -192,6 +194,22 @@ def datatypes(self):
         """
         return self.server_version >= versions[2.0]
 
+    def preflists(self):
+        """
+        Whether bucket/key preflists are supported.
+
+        :rtype: bool
+        """
+        return self.server_version >= versions[2.1]
```

Comment:
That's a great idea.  I just tried this with 2.1.2rc3 but it seems to report back a version of 2.1.1.  It occurred to me that buildbot also is trying to apply the permission in the configuration step, but that is failing.  So putting a check here won't help that.